src/base/static/components/molecules/user-menu.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { translate } from "react-i18next";
+import { Link } from "../atoms/typography";
+import classNames from "classnames";
+import LegacyUtil from "../../js/utils.js";
+
+class UserMenu extends React.Component {
+  state = {
+    isMenuOpen: false,
+  };
+
+  toggleMenu = () => {
+    this.setState(prevState => {
+      return {
+        ...prevState,
+        isMenuOpen: !prevState.isMenuOpen,
+      };
+    });
+  };
+
+  render() {
+    const {
+      slug: datasetSlug,
```

Comment:
If I run this code on a flavor without a `dataset_download` section in the config, I get an error:
```
user-menu.js:82 Uncaught TypeError: Cannot read property 'slug' of undefined
```
Maybe we should have a default value for the `datasetDownloadConfig` of an empty object.